source/index.ts

```diff
@@ -224,7 +303,7 @@ class Conf {
 		}
 	}
 
-	_containsReservedKey(key) {
+	_containsReservedKey(key: string | object): boolean {
```

Comment:
```suggestion
	_containsReservedKey(key: string | {[key: string]: unknown}): boolean {
```

Reply:
Done it manually, had to update other places :)